flair/datasets/sequence_labeling.py

```diff
@@ -4180,19 +4182,25 @@ def __init__(
             }
         }
 
+        # v2.0 only adds new language and splits for AJMC dataset
+        hipe_available_splits["v2.0"] = hipe_available_splits.get("v1.0").copy()
```

Comment:
@stefan-it mypy complains due to this line, causing the unit tests to fail. The 'error' is printed at the end of the test output: 

```
mypy exited with status 1.
_____________________ flair/datasets/sequence_labeling.py ______________________
4186: error: Item "None" of "Optional[Dict[str, Dict[str, List[str]]]]" has no attribute "copy"
===================================== mypy =====================================
```

It seems there is some problem with the `copy()` here. Perhaps it can be removed?

Reply:
Ah, thanks! `copy` is necessary here, because otherwise v1.0 values would be changed, when changing v2.0 variables. The `.get()` method returns an optional causing `mypy` to fail here, so I used the normal index access now. CI is green then :hugs: